app-i18n/opencc/opencc-1.1.7.ebuild

```diff
@@ -14,18 +18,27 @@ S="${WORKDIR}/OpenCC-ver.${PV}"
 
 LICENSE="Apache-2.0"
 SLOT="0/1.1"
-KEYWORDS="~amd64 ~arm64 ~hppa ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
-IUSE="doc test"
+KEYWORDS="~amd64 ~arm64 ~ppc ~ppc64 ~riscv ~x86"
```

Comment:
There should be no reason to drop part of the keyword here.

Reply:
app-i18n/opencc[python] has app-admin/chrpath BDEPEND.

`KEYWORDS` of app-admin/chrpath:

`KEYWORDS="amd64 arm ~arm64 ~mips ppc ppc64 ~riscv x86 ~amd64-linux ~x86-linux ~x64-solaris"`

[Devmanual](https://devmanual.gentoo.org/keywording/#equal-visibility-requirement):

> An ebuild must not depend upon any package that is of a lower keyword level than itself. For example, if foo-1.2 depends upon bar-1.2, and bar-1.2 is ~x86, then foo-1.2 must not be marked stable on x86 unless bar-1.2 is also stabilised.

So ~hppa ~loong are dropped.

Reply:
given these keywords are dropped, I would like to see a revision bump (I can handle this while merging)

can you also file a keyword request for these ARCHs?

Reply:
See https://bugs.gentoo.org/938577.